pkg/apis/hive/v1/agent/platform.go

```diff
@@ -0,0 +1,14 @@
+package agent
+
+// Platform defines agent based install configuration specific to bare metal clusters.
```

Comment:
this comment ties the generically named agent.Platform to bare metal. that doesn't sound right..

Reply:
I guess in this case it should be bare metal platform, as we're not repeating a package name.